lib/pymedphys/tests/mosaiq/test_can_we_db.py

```diff
@@ -0,0 +1,15 @@
+import pymssql
+
+
+@pytest.mark.msq_db
```

Comment:
For the mark `msq_db` to work, need to add something like:

```python
    "msq_db": {
        "options": ["--msq_db"],
        "help": "run test that depend on a Mosaiq Database",
        "description": "mark test as needing a Mosaiq Database",
        "skip_otherwise": True,
    },
```

To the following options:

https://github.com/pymedphys/pymedphys/blob/7011896e99d51926e04befdde1673f871c0d693d/lib/pymedphys/conftest.py#L8-L33